themes/classic/_dev/css/components/imageslider.scss

```diff
@@ -22,6 +22,28 @@
       max-width: 340px;
       .caption-description p {
         color: white;
+        @media (max-width: 767px) {
+          font-size: .875rem;
+        }
+        @media (max-width: 480px){
+          display:none;
+        }
+      }
+      .display-1{
+        @media (max-width: 767px) {
+          font-size: 1.875rem;
+        }
+        @media (max-width: 480px){
+          font-size: 1.5rem;
+        }
+      }
+      h3{
+        @media (max-width: 767px) {
+          font-size: 1rem;
+        }
+        @media (max-width: 480px){
```

Comment:
```suggestion
        @include media-breakpoint-down(sm) {
```

Reply:
When looking at https://getbootstrap.com/docs/4.0/layout/overview/#responsive-breakpoints, isn't it supposed to be 

```
@include media-breakpoint-down(xs) {
```

?